torch/autograd/variable.py

```diff
@@ -4,16 +4,14 @@
 import torch.sparse as sparse
 import torch.utils.hooks as hooks
 
-from ._functions import *
```

Comment:
It is needed, but it can't be done here because of the circular dependencies (actually I might have fixed them now, I'll need to check). I just moved the import to the bottom of this file.